keras_hub/src/models/gemma3/gemma3_decoder_block.py

```diff
@@ -14,16 +14,17 @@
 class Gemma3DecoderBlock(keras.layers.Layer):
     """Transformer decoder layer for Gemma3.
 
-    This is different from Gemma and Gemma2 in several ways:
-
-    - `use_query_key_norm`: Applies RMS Norm on query, key.
-    - `rope_wavelength`: RoPE wavelength differs from local to global attention
-      layers.
-    - `rope_scaling_factor`: RoPE scaling factor differs from local to global
-      attention layers.
-    - `gate_dim_reduction`: In the gating layers, Gemma and Gemma2 reduce
-      intermediate dimension by 2. For Gemma3, no such reduction happens.
-    - Uses bidirectional attention for images, and causal for everything else.
+    This decoder layer is the same as the layer used for Gemma and Gemma2.
+    However, there are a few key differences. Firstly, image tokens have
+    bidirectional masking. Additionally, this layer exposes the following args:
+
+    `use_query_key_norm`: bool. If True, apply RMS normalization on query
```

Comment:
We don't backtick args generally i think

Reply:
Yeah, I backticked these, because I did not put them inside `args`. These are just additional args.